### PR TITLE
Feature/patch user cci 34

### PR DESCRIPTION
**Pull request by mstaron17**

### CODE REVIEW

File: client/public.go [hunk 1]
```diff
@@ -454,3 +454,293 @@ func (p *Profile) UserExistsForEmail(ctx context.Context, token string, email st
...
+		return false, err
+	}
+	data, _ := ioutil.ReadAll(response.Body)
+	defer response.Body.Close()
+
+	if response.StatusCode == http.StatusNotFound {
+		return false, nil
+	}
+
+	if response.StatusCode != http.StatusOK {
```
Comment (flinnb) [inline client/public.go:482]: We should probably log this.  Like:
```golang
		logger := velacontext.GetContextLogger(ctx)
		logger.Info("Get profile error", zap.Any("response", data))
```

File: client/public.go [hunk 2]
```diff
@@ -454,3 +454,293 @@ func (p *Profile) UserExistsForEmail(ctx context.Context, token string, email st
...
+		return err
+	}
+	var dat map[string]interface{}
+	data, _ := ioutil.ReadAll(response.Body)
+	if err = json.Unmarshal(data, &dat); err != nil {
+		return err
+	}
+	if response.StatusCode != http.StatusOK {
+		logger := velacontext.GetContextLogger(ctx)
+		logger.Info("Create profile error", zap.Any("response", dat))
```
Comment (flinnb) [inline client/public.go]: `Patch profile error`?

**Review by flinnb — CHANGES_REQUESTED** [review]
You should probably log non-standard responses in more places.  I marked one, but there seems to be others also.

I have updated all the responses I see

File: client/public.go (hunk 2 above)
Comment (mstaron17) [reply client/public.go]: fixed

**Review by flinnb — APPROVED** [review]
:shipit:

**Pull request merged**